coordinator/src/main/scala/com/socrata/datacoordinator/service/Main.scala

```diff
@@ -85,7 +85,7 @@ class Main(common: SoQLCommon, serviceConfig: ServiceConfig) {
                 u.secondaryManifest.addDataset(storeId, datasetId)
                 // great, now do collocation stuff...
                 val collocatedDatasets =
-                  if (coordinator.secondaryGroupConfigs(storeId).respectsCollocation)
+                  if (coordinator.secondaryGroupConfigs(storeGroup).respectsCollocation)
```

Comment:
This is the crux of it. We needed to pass the store group through the stack to here.